source/marc/terms.ttl

```diff
@@ -355,7 +355,10 @@ marc:publicNote a owl:DatatypeProperty ;
                        marc:TextualHoldingsBasicBibliographicUnit,
                        kbv:Document .
 
-
+# BIB 256
+marc:ComputerFileCharacteristics a owl:Class ;
```

Comment:
I'd prefer the pattern we use for marc notes, i.e. ending with `Note`, but it isn't crucial (as in I don't expect this symbol to take on another meaning, separate from a kind of note).